static/js/components/dashboard/CourseDescription.js

```diff
@@ -21,12 +22,13 @@ const edxLinkBase = `${SETTINGS.edx_base_url}/courses/`;
 
 export default class CourseDescription extends React.Component {
   props: {
-    course: Course
+    courseRun: CourseRun,
+    courseTitle: ?string
   };
 
   needsEdxLink(run: CourseRun): boolean {
-    // Any status besides 'offered' implies that the user has at some point enrolled 
-    return run.status && run.status !== STATUS_OFFERED;
+    // Any status besides 'offered' implies that the user has at some point enrolled
+    return run && run.status !== STATUS_OFFERED;
```

Comment:
actually i just need to get rid of this function entirely. it's not needed anymore per rob's comment